scipy-tutorial/06_xarray_and_dask.ipynb

```diff
@@ -17,9 +17,58 @@
   },
```

Comment:
<p>Can you add local hyperlinks to the h2 sub-headings (<code>##</code> ) below?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>